corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -271,30 +267,16 @@ def fire_change_processed_event(self, change, context):
 
 def handle_pillow_error(pillow, change, exception):
     from pillow_retry.models import PillowError
-    error_id = e = None
+
+    datadog_counter('commcare.change_feed.changes.exceptions', tags=[
+        'pillow_name:{}'.format(pillow.get_name()),
+    ])
 
     # keep track of error attempt count
     change.increment_attempt_count()
 
     # always retry document missing errors, because the error is likely with couch
     if pillow.retry_errors or isinstance(exception, DocumentMissingError):
-        try:
-            error = PillowError.get_or_create(change, pillow)
-        except (DatabaseError, InterfaceError) as e:
-            error_id = 'PillowError.get_or_create failed'
-        else:
-            error.add_attempt(exception, sys.exc_info()[2], change.metadata)
-            error.save()
-            error_id = error.id
-
-    pillow_logging.exception(
```

Comment:
This is what logs it to our log files right? I have used these before. I think it's a bit easier to find old errors with grep than our current UI, but don't have a great reason to keep them now. We will lose any historical errors though because they're deleted from PillowError

Reply:
added back file logging: aedef1d